### PR TITLE
Add new issue and PR templates

### PR DESCRIPTION
**Pull request by cw118**
**This should be merged with mdn/yari#7165.**

This PR proposes adding new templates for issues and PRs (copied from `mdn/content`). Pinging @mdn/localization-team-leads for discussion/reviews :)

### CODE REVIEW

**Comment by caugner** [comment]
Good initiative!

Once this is agreed upon and merged, we should adjust the "Report a problem with this content on GitHub" link" accordingly (see [here](https://github.com/mdn/yari/blob/d1d02fb55525b758781105e1e00bb9b65bed8bc3/client/src/document/on-github.tsx#L112-L129)).

**Review by caugner — APPROVED** [review]
LGTM, but let's wait for approval by the other team leads. 👍

**Comment by SphinxKnight** [comment]
Absolute 💯 here :)

**Review by SphinxKnight — APPROVED** [review]

**Comment by JuanVqz** [comment]
Good call out! 🚀

Comment + question

I've seen you can open different types of issues I been thinking on use that option to use it per language.

Is it possible?

**Review by JuanVqz — APPROVED** [review]
Wow! I haven't use it this way before, thanks! 🚀

File: .github/ISSUE_TEMPLATE/content-bug.yml
Comment (yin1999) [inline .github/ISSUE_TEMPLATE/content-bug.yml:14]: An suggestion: to show localized page, we should remove the language param in URL query:

```suggestion
        🙋 To get help with [Firefox](https://support.mozilla.org/kb/file-bug-report-or-feature-request-mozilla), [Chrome](https://support.google.com/chrome/answer/95315?ref_topic=7439544), [Safari](https://www.apple.com/feedback/safari.html), or another browser, check the browser's support site.
```

But I'm not really sure about the link to Firefox, it just works for me (zh-CN).

File: .github/ISSUE_TEMPLATE/page-report.yml
Comment (yin1999) [inline .github/ISSUE_TEMPLATE/page-report.yml:17]:
```suggestion
        🙋 To get help with [Firefox](https://support.mozilla.org/kb/file-bug-report-or-feature-request-mozilla), [Chrome](https://support.google.com/chrome/answer/95315?ref_topic=7439544), [Safari](https://www.apple.com/feedback/safari.html), or another browser, check the browser's support site.
```

**Review by yin1999 — COMMENTED** [review]
LGTM for l10n-zh, just some suggestions

**Review by hochan222 — APPROVED** [review]
LGTM for `ko-locale` :)

**Comment by cw118** [comment]
Thank you @schalkneethling and everyone for the reviews! 

(@JuanVqz I will merge this one for now, but I think we could create an issue or a discussion to discuss translating issue templates?)

**Pull request merged**
